hw01/src/main/scala/ru/otus/scala/call_count/model/CallCountRequest.scala

```diff
@@ -0,0 +1,3 @@
+package ru.otus.scala.call_count.model
+
+case class CallCountRequest()
```

Comment:
```suggestion
case object CallCountRequest
```

Класс без членов это синглтон.